src/Base/CMakeLists.txt

```diff
@@ -30,6 +30,16 @@ if(MSVC)
         Rpcrt4.lib
         version.lib
     )
+
+    #Universal C runtime introduced in VS 2015 (cl version 19)
```

Comment:
It's the wrong syntax. It should be 
`if (NOT CMAKE_CXX_COMPILER_VERSION VERSION_LESS "19")`